### PR TITLE
jira integration, ticket will be moved to various states if user thum…

### PR DESCRIPTION
**Pull request by mkoryak**
…bs up their PR

### CODE REVIEW

File: src/magic-merge.js [hunk 1]
```diff
@@ -350,6 +380,8 @@ export default class extends EventEmitter {
                             this.emit('warning', `pr #${pr.number} in [${repo}] could not be merged: ${JSON.stringify(notMergable)}`);
                         }
                     } else {
+                        this.updateJiraTicketProgress(queue, pr, 'Code Complete');
```
Comment (bathos) [inline src/magic-merge.js]: Cod*

File: src/magic-merge.js [hunk 2]
```diff
@@ -240,13 +273,10 @@ export default class extends EventEmitter {
                     }
                 }
 
-                if (pr.user.login === 'acconrad') {
-                    // adam's prs are always so sexy
-                    this.addConditionalComment(queue, 'hooray', `Sexy!`);
+                if (ticket) {
+                    this.addConditionalComment(queue, 'hooray', `Jira: [${ticket}](https://${this.settings.jira.host}/browse/${ticket})`);
```
Comment (bathos) [inline src/magic-merge.js]: hoorj*

File: src/magic-merge.js [hunk 3]
```diff
@@ -216,6 +246,9 @@ export default class extends EventEmitter {
 
             prs.forEach(async pr => {
 
+                const prName = pr.head.label.split(':')[1];
+                const ticket = prName.match(/(CAT-\d+)/i) && RegExp.$1 && RegExp.$1.toUpperCase();
+
```
Comment (bathos) [inline src/magic-merge.js]: you do global annex B RegExp stuff just to disturb me I think

Comment (mkoryak) [inline src/magic-merge.js]: it works!

**Pull request merged**